GROOMME.md

```diff
@@ -0,0 +1 @@
+Please groom the changelog and then delete me.
```

Comment:
I believe we should delete this file after groom the changelog

Reply:
So the issue here is that most of the image gallery PRs happened before the 1.8.0 release. So they actually weren't included in this file. After talking with Josh, he suggested just to include this one as you could use the feature flag to identify the code related to the feature.

Reply:
I have brought this up as a topic to discuss for the release retro.